redis-master/attributes/default/default.rb

```diff
@@ -0,0 +1,7 @@
+default['redis-master']['tcp-value'] = "60"
+default['redis-master']['mpsa-value'] = Chef::EncryptedDataBagItem.load('Redis', 'redis', 'redis')['password']
```

Comment:
are you using this attribute?

Reply:
Yes, Used all the attributes mentioned in attributes/default.rb file